packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -208,4 +211,64 @@ export class MyAlgoWalletSession {
 		log("confirmedTx: ", confirmedTx);
 		return confirmedTx;
 	}
+
+	/**
+	 * Creates an algosdk.Transaction object based on execParams and suggestedParams
+	 * @param execParams execParams containing all txn info
+	 * @param txParams suggestedParams object
+	 * @returns array of algosdk.Transaction objects
+	 */
+	makeTx(execParams: ExecParams[], txParams: algosdk.SuggestedParams): Transaction[] {
+		const txns: Transaction[] = [];
+		for (const [_, txn] of execParams.entries()) {
+			txns.push(mkTransaction(txn, txParams));
+		}
+		return txns;
+	}
+
+	/**
+	 * Signs a Transaction object with with myAlgoWallet
+	 * @param transaction transaction object.
+	 * @returns SignedTransaction
+	 */
+	async signTx(transaction: algosdk.Transaction): Promise<SignedTransaction> {
```

Comment:
why should we have `signTx` since we already have `signtransaction`?

Reply:
Again these functions have slightly different interface.